yaml/overlays/dev/calico-vpp-daemonset-dev-patch-nohuge.yaml

```diff
@@ -0,0 +1,68 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: calico-vpp-node
+  namespace: calico-vpp-dataplane
+spec:
+  template:
+    spec:
+      containers:
+        - name: agent
+          image: ${CALICO_AGENT_IMAGE}
+          imagePullPolicy: ${IMAGE_PULL_POLICY}
+          envFrom:
+          - configMapRef:
+              name: calico-agent-dev-config
+          env:
+            - name: DATASTORE_TYPE
+              value: "kubernetes"
+            - name: WAIT_FOR_DATASTORE
+              value: "true"
+            - name: NODENAME
+              value: ""
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+        - name: vpp
+          image: ${CALICO_VPP_IMAGE}
+          imagePullPolicy: ${IMAGE_PULL_POLICY}
+          envFrom:
+          - configMapRef:
+              name: vpp-dev-config
+          env:
+            - name: DATASTORE_TYPE
+              value: "kubernetes"
+            - name: WAIT_FOR_DATASTORE
+              value: "true"
+            - name: NODENAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: LD_LIBRARY_PATH
+              value: "/repo/vpp-manager/vpp_build/build-root/install-vpp-native/vpp/"
+          resources:
+            requests:
+              cpu: 1
+              memory: 4Gi
+            limits:
+              memory: 80Gi
+          volumeMounts:
+            # Add your home to the container we can launch rebulid vpp
+            # without the whole container
+            - name: repo-directory
+              mountPath: /repo
+            - name: ssl-certs
+              mountPath: /etc/ssl/certs/
+            - name: share-certs
+              mountPath: /usr/share/ca-certificates
+      volumes:
+        - name: repo-directory
+          hostPath:
+            path: ${REPO_DIRECTORY}
+        - name: ssl-certs
+          hostPath:
+            path: /etc/ssl/certs/
+        - name: share-certs
+          hostPath:
+            path: /usr/share/ca-certificates
```

Comment:
If we are doing a patchmerge, we can probably leave out this parts if they do not change